R/package.R

```diff
@@ -331,12 +331,19 @@ package_vignettes <- function(path = ".") {
   out[order(basename(out$file_out)), ]
 }
 
-find_template_config <- function(package, bs_version = NULL) {
+find_template_config <- function(package,
```

Comment:
This is called from `as_pkgdown()`, so will be triggered early in all the `build_*` functions, and the call will point to a user facing function